src/dodal/devices/electron_analyser/vgscienta/driver_io.py

```diff
@@ -49,13 +49,16 @@ def __init__(
     ) -> None:
         with self.add_children_as_readables(StandardReadableFormat.CONFIG_SIGNAL):
             # Used for setting up region data acquisition.
-            self.centre_energy = epics_signal_rw(float, prefix + "CENTRE_ENERGY")
-            self.first_x_channel = epics_signal_rw(int, prefix + "MinX")
-            self.first_y_channel = epics_signal_rw(int, prefix + "MinY")
-            self.x_channel_size = epics_signal_rw(int, prefix + "SizeX")
-            self.y_channel_size = epics_signal_rw(int, prefix + "SizeY")
             self.detector_mode = epics_signal_rw(DetectorMode, prefix + "DETECTOR_MODE")
 
+            self.region_min_x = epics_signal_rw(int, prefix + "MinX")
+            self.region_size_x = epics_signal_rw(int, prefix + "SizeX")
+            self.sensor_max_size_x = epics_signal_rw(int, prefix + "MaxSizeX")
```

Comment:
Should: This doesn't connect if I do `dodal connect i09` are we waiting on EPICS changes for this?